src/main/java/org/apache/maven/plugins/shade/mojo/ShadeMojo.java

```diff
@@ -964,11 +992,16 @@ private List<Filter> getFilters()
 
         if ( minimizeJar )
         {
-            getLog().info( "Minimizing jar " + project.getArtifact() );
+            if ( entryPoints == null )
+            {
+                entryPoints = new HashSet<>();
+            }
+            getLog().info( "Minimizing jar " + project.getArtifact()
+                    + ( entryPoints.isEmpty() ? "" : ", entry points = " + entryPoints ) );
```

Comment:
I think this is redudant since other parameters aren't logged at info as well and those who want to see all params this is possible in verbose mode.

Reply:
So I should not log any info at all about the minimisation step, or just not the entry points? I always like to see what is going on in a Maven build, and the information in debug mode is somewhat overwhelming. But if you tell me how you like it or how it is customary, I shall comply.

Reply:
I think it is enough to say that you are going to create a minimized JAR with entry points. We don't log any other verbose input at info level.

Reply:
Which I did in the new commit already.